corehq/apps/app_manager/templates/app_manager/form_designer.html

```diff
@@ -96,7 +96,8 @@
                     return $('header').outerHeight() + 
                         $('.hq-page-header-container').outerHeight();
                 }
-            }
+            },
+            csrftoken: $.cookie('csrftoken'),
```

Comment:
The form builder save request originates in Vellum, so I am passing the CSRF token to vellum when we instantiate the form builder. Vellum handling of this was added in https://github.com/dimagi/Vellum/pull/578